.github/workflows/build.yml

```diff
@@ -6,13 +6,14 @@ name: CI
 jobs:
   flatpak-builder:
     name: "Flatpak Builder"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     container:
-      image: bilelmoussaoui/flatpak-github-actions:gnome-3-38
+      image: bilelmoussaoui/flatpak-github-actions:gnome-3.38
```

Comment:
:')

Reply:
?

Reply:
I spent an hour wondering why this doesn't work and all I had to do is fix one symbol. I find it extremely hilarious 😄

Reply:
Ah it was a mistake on our side :P

Reply:
(the docs were wrong)